src/redux/paymentDetailsSlice.js

```diff
@@ -0,0 +1,45 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { getPaymentDetails } from '../api';
+
+const initialState = {
+  paymentDetails: [],
+  paymentDetailsLoading: true,
+  paymentDetailsError: [],
+  activeOrBoundPaymentDetails: []
+};
+
+export const fetchPaymentDetails = createAsyncThunk(
+  'paymentDetails/paymentDetails',
```

Comment:
In this slice we won't do anything more with `paymentDetails` so I would simplify the name to just `paymentDetails`